src/secret_manager.js

```diff
@@ -0,0 +1,23 @@
+const utils = require("./utils");
+const context = require("./context");
+
+function setSecret(secret_key, secret_val){
+    let process = context.getProcess();
```

Comment:
For any of these variables, if we do not plan on updating the value lets use the `const` keyword